frontend/src/metabase/common/hooks/use-database-candidate-list-query/use-database-candidate-list-query.unit.spec.tsx

```diff
@@ -29,6 +31,18 @@ const TestComponent = () => {
       {data.map((item, index) => (
         <div key={index}>{item.schema}</div>
       ))}
+
+      <div data-testid="metadata">
```

Comment:
I think metadata can only come in search queries, I wouldn't test it here because it can never happen